wolfcrypt/test/test.c

```diff
@@ -2080,7 +2080,7 @@ int poly1305_test(void)
     byte     tag[16];
     Poly1305 enc;
 
-    const byte msg[] =
+    static const byte msg[] =
```

Comment:
That makes sense. Thank you for the explanation Kojo-san.

I only hesitated because the variables will now live for the life of the application but I see we are doing the same in the other test cases so it can't be impacting too much as we often run these on embedded devices.